internal/manifest/constants.go

```diff
@@ -29,6 +29,20 @@ func DefaultAKS() Manifest {
 	return mnfst
 }
 
+// DefaultBaremetalSNP returns a minimal manifest for SNP on baremetal.
+func DefaultBaremetalSNP() Manifest {
+	mnfst := Default()
+	mnfst.ReferenceValues.SNP = SNPReferenceValues{
+		MinimumTCB: SNPTCB{
+			BootloaderVersion: toPtr(SVN(0)),
+			TEEVersion:        toPtr(SVN(0)),
+			SNPVersion:        toPtr(SVN(0)),
+			MicrocodeVersion:  toPtr(SVN(0)),
```

Comment:
IIRC we specifically enabled that these values can be null so we don't write zero SVNs for bare metal.